srcStatic/WindowsModule.h

```diff
@@ -10,5 +11,5 @@
 #endif
 
 
-// Finds the module which contains the given address
 std::string FindModuleName(const void* address);
+HMODULE FindModuleHandle(const void* address);
```

Comment:
Should this return a WindowsUniqueModule instead of an HModule?

Reply:
No, because that would denote a transfer of ownership.